src/views/AccountDetails/AccountDetails.js

```diff
@@ -315,6 +319,97 @@ class AccountDetails extends React.Component {
     history.push({ pathname: `/users/${params.id}/accounts/${status}` });
   };
 
+  getActionMenu = () => () => {
+    const {
+      account,
+      resources,
+      itemDetails
+    } = this.props;
+
+    const feeFineActions = get(resources, ['feefineactions', 'records'], []);
```

Comment:
@iselagb we don't need `get` or `_.get`. Please use optional chaining operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
@mkuklis the changes are already